### ISSUE TITLE
EGARCH with StudentT distribution - Strange summary() output

### ISSUE THREAD

**Issue by xxipei** [bashtage/arch#153]
Hi, Bashtage:

I obtained the following output when running 

model = arch_model(y, p = 1, o= 1, q = 1, vol='EGARCH', power=2.0, dist='StudentT', hold_back=None)

The time series has mean very close to 0. The optimization terminated with successful convergence, but the estimated coefficients look strange. In particular, 

R-squared:  -63807790369.565
Log-Likelihood: -9228.30
AIC: 18468.6
mu: -225.7893 

Optimization terminated successfully.    (Exit mode 0)
            Current function value: 9228.30379484
            Iterations: 34
            Function evaluations: 404
            Gradient evaluations: 33
                        Constant Mean - EGARCH Model Results                        
================================================================
Dep. Variable:               GEM3S_EARNYILD   R-squared:            -63807790369.565
Mean Model:                   Constant Mean   Adj. R-squared:       -63807790369.565
Vol Model:                           EGARCH   Log-Likelihood:               -9228.30
Distribution:      Standardized Student's t   AIC:                           18468.6
Method:                  Maximum Likelihood   BIC:                           18496.6
                                              No. Observations:                  783
Date:                      Thu, Jan 19 2017   Df Residuals:                      777
Time:                              12:36:37   Df Model:                            6
                                  Mean Model                                  
================================================================
                 coef    std err          t      P>|t|        95.0% Conf. Int.
------------------------------------------------------------------------------
mu          -225.7893  3.369e-02  -6701.832      0.000 [-2.259e+02,-2.257e+02]
                               Volatility Model                               
================================================================
                 coef    std err          t      P>|t|        95.0% Conf. Int.
------------------------------------------------------------------------------
omega        -4.8296  3.453e-03  -1398.694      0.000       [ -4.836, -4.823]
alpha[1]       5.7902  2.325e-03   2489.986      0.000       [  5.786,  5.795]
gamma[1]   4.9808  2.000e-03   2489.925      0.000       [  4.977,  4.985]
beta[1]        0.9050  6.899e-05  1.312e+04      0.000       [  0.905,  0.905]
                                 Distribution                                 
================================================================
                 coef    std err          t      P>|t|        95.0% Conf. Int.
------------------------------------------------------------------------------
nu             2.6422  2.694e-02     98.067      0.000       [  2.589,  2.695]
================================================================

Thank you in advance. 
Tracy

**Comment by bashtage** [comment]
Could you share the data you used to estimate the model?

Thanks,
Kevin

**Comment by xxipei** [comment]
[issue data.xlsx](https://github.com/bashtage/arch/files/719829/issue.data.xlsx)

**Comment by bashtage** [comment]
If you scale the returns by 100 it estimates OK.  The bigger problem is that 50 data points is not enough to reliable estimate an ARCH model.  Between 250--500 is the most common minimum number of data points to use.

**Comment by xxipei** [comment]
Strangly, when I use ~700 data as attached, it has the same issue. 
[issue data2.xlsx](https://github.com/bashtage/arch/files/720212/issue.data2.xlsx)

Thank you in advance!

**Comment by bashtage** [comment]
Numerical issues occur when variance are very small.  Multipling you data by 100 works as expected.

```

y  = pd.read_excel('issue.data2.xlsx')
model = arch_model(100 * y, p=1, o=1, q=1, vol='EGARCH',
                   dist='StudentsT', hold_back=None)
res = model.fit()
print(res.summary())

Iteration:      1,   Func. Count:      8,   Neg. LLF: -793.3113982195669
Iteration:      2,   Func. Count:     20,   Neg. LLF: -793.4364171362756
Iteration:      3,   Func. Count:     34,   Neg. LLF: -793.9337385887538
Iteration:      4,   Func. Count:     47,   Neg. LLF: -793.9402579554701
Iteration:      5,   Func. Count:     58,   Neg. LLF: -793.9941859484636
Iteration:      6,   Func. Count:     68,   Neg. LLF: -794.0282761136
Iteration:      7,   Func. Count:     76,   Neg. LLF: -794.2652062104336
Iteration:      8,   Func. Count:     84,   Neg. LLF: -794.3213445395968
Iteration:      9,   Func. Count:     92,   Neg. LLF: -794.5295572351183
Iteration:     10,   Func. Count:    100,   Neg. LLF: -794.7412575867065
Iteration:     11,   Func. Count:    108,   Neg. LLF: -794.8633516253079
Iteration:     12,   Func. Count:    116,   Neg. LLF: -794.905218992873
Iteration:     13,   Func. Count:    124,   Neg. LLF: -794.9196293319771
Iteration:     14,   Func. Count:    132,   Neg. LLF: -794.9242507778312
Iteration:     15,   Func. Count:    140,   Neg. LLF: -794.9247661955695
Iteration:     16,   Func. Count:    148,   Neg. LLF: -794.9248030602678
Iteration:     17,   Func. Count:    158,   Neg. LLF: -794.9248123236794
Optimization terminated successfully.    (Exit mode 0)
            Current function value: -794.924813054
            Iterations: 17
            Function evaluations: 159
            Gradient evaluations: 17
                        Constant Mean - EGARCH Model Results                        
====================================================================================
Dep. Variable:                         data   R-squared:                      -0.000
Mean Model:                   Constant Mean   Adj. R-squared:                 -0.000
Vol Model:                           EGARCH   Log-Likelihood:                794.925
Distribution:      Standardized Student's t   AIC:                          -1577.85
Method:                  Maximum Likelihood   BIC:                          -1549.87
                                              No. Observations:                  783
Date:                      Fri, Jan 20 2017   Df Residuals:                      777
Time:                              18:41:34   Df Model:                            6
                                 Mean Model                                 
============================================================================
                 coef    std err          t      P>|t|      95.0% Conf. Int.
----------------------------------------------------------------------------
mu             0.0238  4.074e-03      5.848  4.989e-09 [1.584e-02,3.181e-02]
                               Volatility Model                              
=============================================================================
                 coef    std err          t      P>|t|       95.0% Conf. Int.
-----------------------------------------------------------------------------
omega         -0.1134      0.346     -0.328      0.743      [ -0.792,  0.565]
alpha[1]       0.0882      0.127      0.697      0.486      [ -0.160,  0.336]
gamma[1]      -0.0161  3.320e-02     -0.485      0.628 [-8.115e-02,4.897e-02]
beta[1]        0.9770  7.122e-02     13.717  7.981e-43      [  0.837,  1.117]
                              Distribution                              
========================================================================
                 coef    std err          t      P>|t|  95.0% Conf. Int.
------------------------------------------------------------------------
nu            23.7134     15.090      1.571      0.116 [ -5.863, 53.289]
========================================================================
```

**Comment by xxipei** [comment]
Thank you very much!

**Comment by xxipei** [comment]
P4: VaR is generally computed for risk horizons of 1-, 5-, and 10-day horizons.
	P4: Maybe a good idea to cite the exact place in Basel III about “increase capital adequacy ratio by 10% ~ 20%”?
	Continuous approximation by Brownian Bridge 
	P8: typo in F ̂_(Y,n) (y) expression: should it be I(Y_j≤y)
	May mention that the backtest procedure for MMVaR can be done in the same way as that for VaR (no additional effort is needed).
	P9: Example 5: intuitively contradictory  must be due to large baseline VaR for positively autocorrelated data
	P19: “a completely different scenario?”  yes at 0.01, but is consistent with 0.05

**Comment by bashtage** [comment]
I think this is closed now.  Please reopen if not.
